blocks/redirects-formatter/redirects-formatter.css

```diff
@@ -0,0 +1,88 @@
+
+:root {
+  --notch-size: 12px;
+}
+
+h1 {
+  margin: 20px;
+  width: fit-content;
+  padding-bottom: 10px;
+  border-bottom: 2px solid black;
+}
+
+textarea {
+  white-space: pre;
+  min-height: 200px;
+  height: fit-content;
+}
+
+button {
+  width: fit-content;
+  grid-column: 4;
+  margin: 16px 0;
+  outline: transparent solid 0;
+  transition: outline 300ms;
+  border: none;
+  height: 36px;
+  font-family: adobe-clean, sans-serif;
+  color: #FFF;
```

Comment:
On line ten you use `black` but here you use the hex code for white. We also have the milo variables (e.g. `--color-white`). I don't really have a preference, it just seems like we should keep it consistent